_sass/components/partners/_partners.scss

```diff
@@ -2,6 +2,7 @@
 
 @use 'uswds-core' as *;
 @use '../../variables' as *;
+@forward 'usa-link';
```

Comment:
Is this `@forward` necessary? I would expect the styles to already be available through [the `uswds` forwarding](https://github.com/GSA-TTS/identity-site/blob/5cf71d4ee90e56ca1a4c7d075dbda0cf5c69d8ad/assets/scss/main.scss#L9), and the mixin usage below is coming from the `uswds-core` import a couple lines above this.

```suggestion
```

Reply:
Without that line an error is produced in the console. Some differences in the SCSS setup between this repo and others will be reviewed.